target/check-for-changes.sh

```diff
@@ -79,8 +79,15 @@ do
         esac
       done
 
-      # regenerate postix aliases
+      # Generate default /etc/aliases
       echo "root: ${PM_ADDRESS}" >/etc/aliases
+      echo "amavis: ${PM_ADDRESS}" >>/etc/aliases
+      if [[ ${ENABLE_CLAMAV} -eq 1 ]]
+      then
+        echo "clamav: ${PM_ADDRESS}" >>/etc/aliases
+      fi
+
+      # Append external postifx aliases
       if [[ -f /tmp/docker-mailserver/postfix-aliases.cf ]]
```

Comment:
This is probably fine, but we could check if postfix-aliases.cf has a mapping for amavis. Many existing installations may already have one (mine does). Not a big deal though, perhaps it is better to keep the code simple and write something about it in the README.

Reply:
Hmmm, mine didn't.   My /tmp/dockermailserver/postfix-aliases.cf is empty.  If that's a better place to do these, then by all means.